### PR TITLE
Merge camunda upgrade  to develop

### PR DESCRIPTION
**Pull request by SnehaSuresh-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-2344

# Changes

* Merging camunda upgrade changes to develop branch
# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Comment by abhilash-aot** [comment]
Please check the conflicts @SnehaSuresh-aot

File: VERSION [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (abhilash-aot) [inline VERSION:1]: Space isnt needed here

Comment (SnehaSuresh-aot) [reply VERSION]: updated @abhilash-aot

File: VERSION [hunk 2]
```diff
@@ -1,3 +1,4 @@
 v5.2.0-alpha
 
 
+
```
Comment (abhilash-aot) [inline VERSION:4]: No need for change here

File: forms-flow-bpm/pom-default.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>org.camunda.bpm.extension</groupId>
         <artifactId>formsflow-bpm</artifactId>
-        <version>5.1.0</version>
+        <version>5.1.1</version>
```
Comment (sumesh-aot) [inline forms-flow-bpm/pom-default.xml:10]: Shouldn't we update the version to 5.2.0 ?

File: forms-flow-bpm/pom.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 	<groupId>org.camunda.bpm.extension</groupId>
 	<artifactId>formsflow-bpm</artifactId>
-	<version>5.1.0</version>
+	<version>5.1.1</version>
```
Comment (sumesh-aot) [inline forms-flow-bpm/pom.xml:9]: Same here

**Review by abhilash-aot — APPROVED** [review]

**Review by auslin-aot — APPROVED** [review]

**Pull request merged**